modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -215,7 +215,7 @@ const Combobox = ({
   }, [autocompleteItems, isFocused, value]);
 
   const handleAutocompleteClick = (
-    event: React.SyntheticEvent,
+    event: React.KeyboardEvent | React.MouseEvent,
```

Comment:
I'm open to ideas about how to better handle this, but because we use this function for keyboard and click events, I went with this typing.

Reply:
This is an internal function. Nothing is being used from the `event` object. This function does call `menuItem.onClick` which is from `MenuItem.tsx` which is expecting a mouse event. Did you run into any issues there?

The external interface is the `onChange` which is a change event on an `input` element. I think we'd be okay if we had a signature like this - either `KeyboardEvent | MouseEvent`. For the external consumer, they would have to do detection if they wanted access to something unique about the event (like which mouse button was down or what key was detected).

Reply:
Yep, I'm fixing those issues now. Having a keyboard and mouse event call the same function is tricky to type.